### DISCUSSION TITLE
Clickable lesson names

### DISCUSSION THREAD

**Discussion by Dinock** [GDQuest/learn-gdscript#140] [Ideas]
It could be helpful if there was the possibility to click in the topmost line on the name of the lesson to open the course index. Right now it is only possible to click on the small symbol in the top left corner to open the course index.

When clicking on the lesson's name the font should also change its colour (see suggestion-mockup).

Clicking (now):
![clicking_before](https://user-images.githubusercontent.com/54438803/149022282-9ed5d4c1-e203-4394-a0bd-dd0ff9337d2a.png)

Clicking (suggestion-mockup):
![clicking_after](https://user-images.githubusercontent.com/54438803/149022339-686d056a-078b-497a-9c61-1d94963cb273.png)

**Comment by YuriSizov** [comment]
Is the button hard to hit?

**Reply by Dinock** [reply]
No, I don't think so.
But to me it feels more natural to be able to click on the lesson's name _as well_ to get the same result.
At least I would expect something to happen if I click on the name.

**Comment by NathanLovato** [comment]
I don't think that'll be logical to users.

Breadcrumbs usually work by taking you to the page corresponding to the title you click. Not by taking you back to the homepage when clicking the current page's title.

Also, the idea with ux is we make changes to solve problems users have. Its when people have trouble accessing the index we should make changes.

**Reply by Dinock** [reply]
Ah, I understand. This sounds reasonable to me. :)
